.drone.yml

```diff
@@ -0,0 +1,24 @@
+env:
+- REPORT_CARD_GITHUB_STATUS_TOKEN=$$report_card_github_status_token
+- REPORT_CARD_GITHUB_REPO_TOKEN=$$report_card_github_repo_token
+- REPORT_CARD_GITHUB_API_TOKEN=$$report_card_github_api_token
+image: clever/drone-go:1.6
+notify:
+  email:
+    recipients:
+    - drone@clever.com
+  slack:
+    on_failure: true
+    on_started: false
+    on_success: false
+    webhook_url: $$slack_webhook
+script:
+- sudo pip install -q git+https://$REPORT_CARD_GITHUB_REPO_TOKEN@github.com/Clever/report-card.git; GITHUB_API_TOKEN=$REPORT_CARD_GITHUB_STATUS_TOKEN report-card --publish || true
+- pip install -q git+git://github.com/Clever/report-card.git && GITHUB_API_TOKEN=$REPORT_CARD_GITHUB_API_TOKEN report-card --publish
```

Comment:
i think there's a new way of enabling report card as a publish hook cc @nathanleiby